src/ui/screen/screen_setup.cpp

```diff
@@ -221,6 +239,9 @@ bool CScreenSetup::EventProcess(const Event &event)
                 m_main->ChangePhase(PHASE_SETUPss);
                 return false;
 
+            case EVENT_INTERFACE_SETUPm:
```

Comment:
This shouldn't ever happen. Add `assert(false)` to make it more explicit?

Reply:
(or maybe I'm stupid and it should - either way, add a comment why is it different than the others)